spring-cloud-dataflow-rest-resource/src/main/java/org/springframework/cloud/dataflow/rest/resource/TaskExecutionResource.java

```diff
@@ -102,6 +108,7 @@ public TaskExecutionResource() {
 	public TaskExecutionResource(TaskJobExecutionRel taskJobExecutionRel) {
 		Assert.notNull(taskJobExecutionRel, "taskJobExecutionDTO must not be null");
 		this.executionId = taskJobExecutionRel.getTaskExecution().getExecutionId();
+		this.parentExecutionId = taskJobExecutionRel.getTaskExecution().getParentExecutionId();
```

Comment:
Build failed because the parentExecutionId was not present in the Rest API Docs.   Will fix on merge.